apps/client/src/components/addToast/addToast.tsx

```diff
@@ -50,25 +58,42 @@ export const AddToast: React.FC<AddToastProps> = ({
         <div className={styles.addToastForm}>
           <div className={styles.form}>
             <div className={styles.titleAndBox}>
-              <label htmlFor="">username</label>
-              <select
-                className={styles.inputBox}
-                name="user"
-                id="user"
-                onChange={(e) => setUser(e.target.value)}
-              >
-                {!users || users.length === 0 ? (
-                  <p> No user found</p>
-                ) : (
-                  users.map((user: User, index: number) => (
-                    <option key={index} value={user.id}>
-                      {user.username}
-                    </option>
-                  ))
-                )}
-              </select>
+              <label htmlFor="">שם משתמש</label>

```

Comment:
htmlFor needs to refer to an id or name of a select or input component. Research it on google